_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,441 @@
+		</nav>
+	</div>
+</body>
+```
+
+#### Passed Example 14
+
+No [text node][] on this page includes any of the [visual reference words][].
+
+```html
+<p>Click the button, for a surprise</p>
+<button onclick="alert('Surprise!')">Howdy</button>
+```
+
+#### Passed Example 15
+
+The [text node][] inside the `p` element is not applicable because it is neither [visible][] nor [included in the accessibility tree][] due to its `display:none` styling. The [text node][] inside the `button` element does not include any of the [visual reference words][].
+
+```html
+<p style="display:none">Click the box, for a surprise</p>
+<button onclick="alert('Surprise!')">Howdy</button>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This paragraph includes the [visual reference word][] "right". The user is told to find the menu on the right but the menu is not identified in any other way.
+
+```html
+<head>
+	<title>Failed example 1 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the menu on the right, to navigate</p>
+		</div>
+		<div class="col">
+			<ul>
+				<li>
+					<a href="https://www.w3.org/Consortium/contact">Contact</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Help/">Help and FAQ</a>
+				</li>
+			</ul>
+		</div>
+	</div>
+</body>
+```
+
+#### Failed Example 2
+
+This paragraph includes the [visual reference word][] "right". The user is told to find the navigation on the right and the navigation is correctly identified by a `nav` element, but there are 2 `nav` elements on the page so the user doesn't know which one to use (the non-visual reference is ambiguous and does not identify a specific [web content][]).
+
+```html
+<head>
+	<title>Failed example 2 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<nav>
+		<ul>
+			<li>
+				<a href="https://www.w3.org/">W3C homepage</a>
+			</li>
+			<li>
+				<a href="https://www.w3.org/standards/">Standards</a>
+			</li>
+		</ul>
+	</nav>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the navigation on the right, for the non-essential links</p>
+		</div>
+		<nav>
+			<div class="col">
+				<ul>
+					<li>
+						<a href="https://www.w3.org/Consortium/contact">Contact</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Help/">Help and FAQ</a>
+					</li>
+				</ul>
+			</div>
+		</nav>
+	</div>
+</body>
+```
+
+#### Failed Example 3
+
+This paragraph includes the [visual reference word][] "triangle" (which is not included in the identified content) and no other indication is present so the rule fails. The fact that the triangle menu is on a different page of the same website does not restrict the rule.
+
+```html
+<body>
+	<p>
+		On the <a href="/test-assets/SC1.3.3-triangle-menu-without-heading.html">information page</a> you can find more
+		examples within the triangle menu
```

Comment:
It is actually passed as the instruction does not state that anything about the triangle menu is without headings or with headings. The instruction in the passed example 6 and failed example 3 example the link if we only consider the instructional text then it fails passed example 6 as it does not specify that triangle menu is with or without headings.
If the instruction is following than it failed the rule (you need to update the passed example 6 accordingly):
```suggestion
		examples within the heading 'Triangle Menu'
```
Then you can say that there is no such heading 'Triangle Menu' in the identified content.

Reply:
There is a triangle menu in the content, it is identified with the ▲ symbol (a sensory characteristic, hence bad).
In Passed Example 6, the menu is identified with the "Triangle menu" text, a non-sensory characteristic (good).